backend/report_submission/forms.py

```diff
@@ -22,7 +22,6 @@ class GeneralInformationForm(forms.Form):
     auditee_contact_title = forms.CharField()
     auditee_phone = forms.CharField()
     auditee_email = forms.CharField()
-    user_provided_organization_type = forms.CharField()
```

Comment:
this field is not actually submitted as part of the general information form, so we're removing it from the expected fields here